apisix/plugins/proxy-rewrite.lua

```diff
@@ -263,109 +274,118 @@ do
     end
 
 
-function _M.rewrite(conf, ctx)
-    for _, name in ipairs(upstream_names) do
-        if conf[name] then
-            ctx.var[upstream_vars[name]] = conf[name]
+    function _M.rewrite(conf, ctx)
```

Comment:
Don't change the original indent of this file in this PR, which should reflect this feature only.
If necessary, submit another PR to fix it.

Reply:
Files are automatically formatted by vscode editor. I'll revert changes except code change related to this feature only.